README.md

```diff
@@ -1,16 +1,144 @@
-# Title
+# Validating Pseudocode
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Identify a process to validate pseudocode
 
 ## Introduction
 
-## SWBAT 1
+Let's review the pseudocode we produced previously. Did you find a bug in it?
+Chances are you did. Don't worry if it didn't leap out for you. Our process will
+help you spot the issue. Let's analyze our process for validating our
+pseudocode, the 6<sup>th</sup> step in the Flatiron Programming Process.
 
-## SWBAT 2
+## Identify a Process to Validate Pseudocode
+
+Here's the pseudocode again:
+
+```text
+Procedure DivideBaguetteEvenly(baguette, n):
+  baguette_length = measure(baguette)
+  even_length = baguette_length / n
+  collection = []
+
+  while baguette_length > even_length:
```

Comment:
I think it is written >= in the previous lesson

Reply:
other way around, I realized at the end - last lesson has >= when it should have >